.config/.cprc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "version": "4.12.0"
```

Comment:
just curious what this does if you happen to know?

Reply:
It was created when I ran `npx @grafana/create-plugin@latest update`. It's the version of `create-plugin`  that was used. Not sure what it's used for exactly though.